File: 1635-number-of-good-pairs/number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        
        store = {}
        ans = 0
        for num in nums:
            if num in store:
                store[num] += 1

            else:
                store[num] = 1

        for val in store.values():
            if val >= 2:
                ans += (val * (val - 1)) / 2

        return int(ans)

File: 1635-number-of-good-pairs/number-of-good-pairs.ts
function numIdenticalPairs(nums: number[]): number {
    let count: number = 0;
    for(let i = 0; i < nums.length; i++){
        for(let j = i + 1; j< nums.length; j++){
            if( nums[i] == nums[j] && i < j){
                count++;
            }
        }
    }
    return count
};